packages/components/unpublished-development-types/global.d.ts

```diff
@@ -9,6 +9,7 @@ import type EmberStargateRegistry from 'ember-stargate/template-registry';
 import type EmberStyleModifierRegistry from 'ember-style-modifier/template-registry';
 import type EmberTruthRegistry from 'ember-truth-helpers/template-registry';
 import type PowerSelectRegistry from 'ember-power-select/template-registry';
+// eslint-disable-next-line ember/no-at-ember-render-modifiers
```

Comment:
question: what's the alternative to `ember-render-modifiers`?

Reply:
we have not found suitable solution but here a resource that talks in more details about this https://nullvoxpopuli.com/avoiding-lifecycle/

Reply:
I was doing some research as well and it seems pretty complicated to migrate off `ember-render-modifiers`.